iglo/league/utils/ogs.py

```diff
@@ -10,3 +11,56 @@ def fetch_sgf(sgf_url: str) -> str:
     if response.status_code != 200:
         raise OGSException("can not fetch SGF file")
     return response.content.decode()
+
+
+def get_player_data(username: str) -> Dict[str, Any]:
+    """
+    Get complete player data from OGS by username.
+    
+    Args:
+        username: The OGS username to look up
+        
+    Returns:
+        A dictionary with player data including ID, rating, and deviation
+        
+    Raises:
+        OGSException: If there's an error communicating with the OGS API
+    """
+    try:
+        # First get the player ID
+        response = requests.get(f"https://online-go.com/api/v1/players?username={username}")
+        if response.status_code != 200:
+            raise OGSException(f"Failed to fetch player data: HTTP {response.status_code}")
```

Comment:
since you catch at the end anyway you can skip this if and raise and use response.raise_for_status() instead

Reply:
Done.